src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/CaseManagementForCaseWorkerServiceTest.java

```diff
@@ -867,34 +867,6 @@ void setEt3ResponseDueDate_Serving_Date_Plus_28() {
         assertEquals(expectedEt3DueDate, caseData.getEt3DueDate());
     }
 
-    @Test
-    void setNextListedDate() {
```

Comment:
Why remove this test? It looks like you've not touched setNextListedDate in the original code